### PR TITLE
Updated dh-psi-monitoring service monitor configurations

### PR DESCRIPTION
**Pull request by HumairAK**

### CODE REVIEW

File: monitoring/psi-prod-cluster/bases/alertmanager/alertmanager.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: aicoe-sre-alertmanager
   namespace: dh-psi-monitoring
 spec:
-  replicas: 3
```
Comment (anishasthana) [inline monitoring/psi-prod-cluster/bases/alertmanager/alertmanager.yaml:7]: Why are we using only 1 replica?

File: monitoring/psi-prod-cluster/bases/prometheus/prometheus.yaml [hunk 2]
```diff
@@ -11,17 +11,14 @@ spec:
       - name: aicoe-sre-alertmanager
         namespace: dh-psi-monitoring
         port: web
-  replicas: 2
```
Comment (anishasthana) [inline monitoring/psi-prod-cluster/bases/prometheus/prometheus.yaml:14]: Similar to above, why remove our HA?

Comment (HumairAK) [reply monitoring/psi-prod-cluster/bases/prometheus/prometheus.yaml:14]: this namespace has very limited resources, putting these to 3 & 2 respectively would almost max out our cpu and memory

Comment (anishasthana) [reply monitoring/psi-prod-cluster/bases/prometheus/prometheus.yaml:14]: Sounds good. I've opened https://github.com/AICoE/idh-manifests/issues/19 so we don't forget. In the meanwhile we can go ahead with the PR

**Review by anishasthana — APPROVED** [review]

**Pull request merged**